cluster-autoscaler/simulator/delta_cluster_snapshot.go

```diff
@@ -353,6 +355,27 @@ func (snapshot *deltaSnapshotNodeLister) Get(nodeName string) (*schedulerframewo
 
 // IsPVCUsedByPods returns if PVC is used by pods
 func (snapshot *deltaSnapshotStorageLister) IsPVCUsedByPods(key string) bool {
+	data := snapshot.data
+	if data.pvcNamespaceMap != nil {
+		if v, found := data.pvcNamespaceMap[key]; found && v > 0 {
+			return true
+		}
+		return false
+	}
+	nodeInfos := snapshot.data.getNodeInfoList()
+	pvcNamespaceMap := make(map[string]int)
+	for _, v := range nodeInfos {
+		for k, i := range v.PVCRefCounts {
+			if _, found := pvcNamespaceMap[k]; !found {
+				pvcNamespaceMap[k] = 0
+			}
+			pvcNamespaceMap[k] += i
+		}
+	}
+	data.pvcNamespaceMap = pvcNamespaceMap
+	if v, found := data.pvcNamespaceMap[key]; found && v > 0 {
```

Comment:
nit: Since zero value for int is 0, you can just do `return data.pvcNamespaceMap[key] > 0` here: it doesn't matter if map key was found or not. Same above, since these lines are duplicated.

Reply:
Done.